specification/web/resource-manager/Microsoft.Web/stable/2024-11-01/CommonDefinitions.json

```diff
@@ -383,6 +383,10 @@
               "description": "If <code>true</code>, this App Service Plan will perform availability zone balancing.\nIf <code>false</code>, this App Service Plan will not perform availability zone balancing.",
               "default": false,
               "type": "boolean"
+            },
+            "asyncScalingEnabled": {
+              "description": "If <code>true</code>, this App Service Plan will attempt to scale asynchronously if there are insufficient workers to scale synchronously.\nIf <code>false</code>, this App Service Plan will only attempt sync scaling.",
+              "default": false
```

Comment:
[ARMBlockingComment] You need to define a type here. 

PLease consider modelling this as an enum instead of boolean. For example : 

AsyncScaling: Enabled\Disabled ,this allows you to keep this extensible later to add new states like Suspended etc